docs/source/algorithms/MaskDetectors-v1.rst

```diff
@@ -6,35 +6,99 @@
 
 .. properties::
 
+Introduction
+------------
+
+To understand the algorithms options, user should clearly understand the difference between *WorkspaceIndex* 
+-- the numbers, specified in *WorkspaceIndexList* and *StartWorkspacIndex*, *EndWorkspaceIndex* properties,
+the *Spectra ID* -- values of the **SpectraList** property and *Detector ID* -- the numbers to provide for 
+*DetectorList* property.
+
+The *WorkspaceIndex* is the number a spectra has in a workspace, e.g. ::
```

Comment:
...and here I would phrase this as 'The *WorkspaceIndex is the index of a spectrum in a workspace'.
Anyway, spectra -> spectrum.